extensions/ql-vscode/src/skeleton-query-wizard.ts

```diff
@@ -0,0 +1,247 @@
+import { join } from "path";
+import { CancellationToken, Uri, workspace, window as Window } from "vscode";
+import { CodeQLCliServer } from "./cli";
+import { OutputChannelLogger } from "./common";
+import { Credentials } from "./common/authentication";
+import { QueryLanguage } from "./common/query-language";
+import { askForLanguage, isFolderAlreadyInWorkspace } from "./helpers";
+import { getErrorMessage } from "./pure/helpers-pure";
+import { QlPackGenerator } from "./qlpack-generator";
+import { DatabaseManager } from "./local-databases";
+import * as databaseFetcher from "./databaseFetcher";
```

Comment:
Can we just import the methods we need here instead of importing the complete module? Right now, I'm not sure why it's different than the other imports.

Reply:
I had to do this in order to [spy on methods for the databaseFetcher](https://github.com/github/vscode-codeql/pull/2250/files#diff-69022948abbb5354a1731c421934f96e1b96b53a2efc045bf40a3c7b78be1ea5R90):

```
downloadGitHubDatabaseSpy = jest
      .spyOn(databaseFetcher, "downloadGitHubDatabase")
      .mockResolvedValue(undefined);
```

I'm not sure if there's a better way.

Reply:
Done here: https://github.com/github/vscode-codeql/pull/2305